lets-merge-client-front/src/store/modules/snackbar.js

```diff
@@ -0,0 +1,25 @@
+export default {
+  namespaced: true,
+  state: {
+    message: "",
+    visible: false,
+  },
+  mutations: {
+    SHOW(state, message) {
+      state.message = message;
+      state.visible = true;
+    },
+    HIDE(state) {
+      state.message = "";
+      state.visible = false;
+    }
+  },
+  getters: {
+    message(state) {
```

Comment:
메소드 네이밍이니까 `getMessage`가 더 낫지 않을까요?
```suggestion
    getMessage(state) {
```

Reply:
아래처럼 메소드 호출 시 get이라는 단어 중복 방지를 위해 저렇게 구현해두었는데요!
`this.$store.getters["snackbar/getMessage"]`
`this.$store.getters["snackbar/message"]`

팀벤션으로 정할만한 사항인 것 같네요. 회의 때 이야기해봅시다 🙂